### PR TITLE
Fix TCPConnection for verify_ssl = false

### LINKED ISSUE #32: aiohttp client constructor is non-functional if verify_ssl=False is used
Currently the library passes both verify_ssl and ssl_context to the [TCPConnector](https://github.com/tomplus/kubernetes_asyncio/blob/master/kubernetes_asyncio/client/rest.py#L67), but the TCPConnector in recent aiohttp versions does not support having both.

```
ValueError: verify_ssl, ssl_context, fingerprint and ssl parameters are mutually exclusive
```

### PR DESCRIPTION
**Pull request by bpicolo**
This fixes #32 

I didn't see any relevant tests but am happy to add any test that you think should be here - unfamiliar with working in this library so I wouldn't know what makes the most sense off hand : )

### CODE REVIEW

File: kubernetes_asyncio/client/rest.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)  # noqa: E501
 
     OpenAPI spec version: v1.10.1
-    
+
```
Comment (tomplus) [inline kubernetes_asyncio/client/rest.py]: Could you rollback this unnecessary change ?

File: kubernetes_asyncio/client/rest.py [hunk 2]
```diff
@@ -66,8 +66,8 @@ def __init__(self, configuration, pools_size=4, maxsize=4):
 
         connector = aiohttp.TCPConnector(
             limit=maxsize,
-            ssl_context=ssl_context,
-            verify_ssl=configuration.verify_ssl
+            ssl_context=ssl_context if configuration.verify_ssl else None,
+            verify_ssl=configuration.verify_ssl,
```
Comment (tomplus) [inline kubernetes_asyncio/client/rest.py]: ... and this `,` too. Small and precise change-set is simpler to read and apply. Thank you.

File: kubernetes_asyncio/client/rest.py (hunk 1 above)
Comment (bpicolo) [reply kubernetes_asyncio/client/rest.py]: hah sure, that was just editor auto trimming trailing whitespace

**Comment by bpicolo** [comment]
I must admit I totally missed that comment. Will also go upstream in swagger codegen - thanks for being open to the PR nevertheless

**Comment by tomplus** [comment]
LGTM
